runtime/caml/misc.h

```diff
@@ -29,7 +29,7 @@
 
 #include "camlatomic.h"
 
-/* Detection of available C attributes */
+/* Detection of available C compiler extensions */
```

Comment:
Nitpicking, that would be 
```suggestion
/* Detection of available C attributes and compiler extensions */
```
as attributes are part of the (recent) standard.